snippets.kak

```diff
@@ -8,7 +8,7 @@ hook global WinSetOption 'snippets=$' %{
 hook global WinSetOption 'snippets=.+$' %{
     set window snippets_triggers_regex %sh{
         eval set -- "$kak_opt_snippets"
-        if [ $(($#%3)) -ne 0 ]; then printf '\A\z'; exit; fi
+        if [ $(($#%3)) -ne 0 ]; then printf '\\\A\\\z'; exit; fi
```

Comment:
Is this one necessary as well? I understand that such characters need to be escaped inside `".."`, but I thought `'..'` strings did no escaping at all.

Reply:
I had to check, but it seems like it's needed. I guess it's not the shell that makes problems here, but `printf` itself. Here is what happens:
```
$ printf '\A\z'
printf: unknown escape sequence `\A'
printf: unknown escape sequence `\z'
Az$
```

`printf '%s' '\A\z'` and even `printf "%s" "\A\z"` work without a problem for me.